src/uwtools/cli/run_forecast.py

```diff
@@ -18,13 +19,17 @@ def main() -> None:
     Parses arguments provided by the user and passes to the Forecast driver class to be run.
     """
     args = parse_args(sys.argv[1:])
-    name = "run-forecast"
-    cli_helpers.setup_logging(
+    name = "validate-config"
```

Comment:
Should this still be `run-forecast`?

Reply:
caught my copy-paste rushing errors, thanks